src/uxnrnbbs/nrnbbs.cpp

```diff
@@ -13,7 +13,7 @@
 #include <string.h>
 
 #include <sys/stat.h>
-//extern "C" { extern int mkdir(const char*, int); }
+extern "C" { extern int mkdir(const char*, int); }
```

Comment:
why was this uncommented?

Reply:
I guess I was testing, will remove and see what gives

Reply:
Removed